UKHomeOffice/nmsw-ui
# NMSW-449 Add wording to link on create password page

# Ticket

NMSW-449

----

## AC

Create account - your password - add new wording to the end of 3 random words (opens in new tab)

----

## To test

- Run app
- Go to localhost:3000
- Go to /create-account/your-password
- The link text should read `3 random words (opens in new tab)`

----

## Notes

Also made a small change to the /create-account/email-address heading to include missing question mark
